sal_interfaces/Scheduler/Scheduler_Commands.xml

```diff
@@ -36,4 +36,72 @@
     <EFDB_Topic>Scheduler_command_computePredictedSchedule</EFDB_Topic>
     <Description>This command causes the scheduler to compute and publish the predicted schedule. It can only run if the scheduler is idle (resume command not sent). This runs automatically as part of the advance operation mode so this command is only useful when users want to estimate what is ahead when the scheduler is not running.</Description>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_command_addBlock</EFDB_Topic>
+    <Description>Add new block to the scheduling queue.</Description>
+    <item>
+      <EFDB_Name>id</EFDB_Name>
+      <Description>The name of the block (e.g., OBS-123).</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>1</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>override</EFDB_Name>
+      <Description>Override information in the block with those provided here.</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>1</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_command_removeBlock</EFDB_Topic>
+    <Description>Remove blocks from the scheduling queue.</Description>
+    <item>
+      <EFDB_Name>ids</EFDB_Name>
+      <Description>Comma-separated list of block ids to be removed (e.g., OBS-123,OBS-124).</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>1</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_command_validateBlock</EFDB_Topic>
+    <Description>Validate block.</Description>
```

Comment:
All blocks get validated when the CSC gets enabled right?
If so, then then the `id` parameter is useless.

How does the CSC see new blocks? Presumably it has to go to Standby and back up, yes?

Reply:
You can ask to validate a particular block with an override.

I think the CSC can monitor the block directory and validate things when they change.

Reply:
ok, not exactly sure how that works as the block directory will be a database. So you think it'll continually poll the database for all tickets marked as "Ready" and have not been validated? 
We can leave this in.. i'm just struggling to see how it fits in the big picture.